docs/UserGuide.md

```diff
@@ -345,22 +321,121 @@ Examples:
 * `c-find John` returns `john` and `John Doe`.
 * `c-find alex david` returns `Alex Yeoh`, `David Li`.<br>
 
-#### 3.7 Locating persons based on matching tags: `c-tag-find`
+#### 3.5 Locating employees based on matching tags: `c-tag-find`
 
-Finds all active(unarchived) contacts that contain the KEYWORD(s) in their tags.
+Finds all employees whose tags contain the one or more `KEYWORD(s)`.
 
-Format: `c-tag-find KEYWORD [MORE_KEYWORDS] ...`
+Format: `c-tag-find KEYWORD [MORE_KEYWORDS]`
 
-* The search is case-insensitive. e.g `Friday` will match `friday`.
-* The order of the keywords does not matter. e.g. `friday monday` will match `monday friday`.
-* Only the tag names are searched.
-* Only full words will be matched e.g. `PartTime` will not match `PartTimes`.
-* Employees whose tag(s) matching at least one keyword will be listed in the `Employee Contact DeatailL` pane
- (i.e. `OR` search).  e.g. `Friday Monday` will return employees whose tags contain `Friday` or `Monday`.
+* The search is case-insensitive. For example, `Friday` will match `friday`.<br>
+* The order of the keywords does not matter. For example, `monday friday` will match `monday friday`.<br>
+* Only the tag names are searched.<br>
+* Only full words will be matched. For example, `Fri` will not match `Friday`.<br>
+* Employees whose tag(s) matching at least one keyword will be listed in the Employee
+ Directory inside the _Main View_ (i.e. `OR` search).  For example, `c-tag-find Monday Friday` will find employees whose
+  tags contain `Monday` or `Friday`.<br>
 
-Examples:
-* `c-tag-find friday` returns all employees whose tags contain `friday`.
-* `c-tag-find friday parttime` returns all employees whose tags contain `friday`, `parttime`.<br>
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Note:**<br>
+* In tCheck, if an employee contains a tag for a specific day, it means this employee is available on that day
```

Comment:
I don't think you need the bullet point here, as there is only one point.

Reply:
Will remove it. Thanks!